### PR TITLE
SSCS-9480: Unable to Archive "Other Benefit" cases on multi-draft page

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/service/converter/ConvertSscsCaseDataIntoSessionDraft.java [hunk 1]
```diff
@@ -406,8 +406,11 @@ private SessionPostcodeChecker buildSessionPostcode() {
     private SessionBenefitType buildSessionBenefitType(BenefitType benefitType) {
         if (benefitType == null) {
             return null;
+        } else if (StringUtils.length(benefitType.getCode()) <= 3) {
+            return new SessionBenefitType(benefitType.getDescription() + " (" + benefitType.getCode() + ")");
+        } else {
+            return new SessionBenefitType(benefitType.getDescription());
```
Comment (srochani) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/converter/ConvertSscsCaseDataIntoSessionDraft.java:412]: This string should match with the benefit types.js file values in FED

**Review by rasikasri — APPROVED** [review]

**Review by jackmaloney — APPROVED** [review]

**Comment by srochani** [comment]
Closing and reopening the PR

**Comment by srochani** [comment]
Closing and reopening the PR

**Comment by srochani** [comment]
Closing the PR and opening new PR

**Comment by srochani** [comment]
Reopening the PR

**Pull request merged**